test/unit/mdc-textfield/foundation.test.js

```diff
@@ -56,32 +56,37 @@ test('defaultAdapter returns a complete adapter implementation', () => {
   ]);
 });
 
-const setupTest = () => {
+const setupTest = ({
+  useCharacterCounter = false,
+  useHelperText = false,
+  useLeadingIcon = false,
+  useTrailingIcon = false,
+} = {}) => {
   const mockAdapter = td.object(MDCTextFieldFoundation.defaultAdapter);
-  const helperText = td.object({
+  const helperText = useHelperText === true ? td.object({
```

Comment:
I think this can just be `useHelperText ?td...`

Reply:
Oh yeah, duh... Fixed